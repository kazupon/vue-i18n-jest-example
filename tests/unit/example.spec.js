import { mount, createLocalVue } from '@vue/test-utils'
import VueI18n from 'vue-i18n'
import HelloI18n from '@/components/HelloI18n.vue'

describe('HelloI18n.vue', () => {
  it('renders i18n custom blocks', () => {
    const localVue = createLocalVue()
    localVue.use(VueI18n)
    const i18n = new VueI18n({
      locale: 'en'
    })
    const wrapper = mount(HelloI18n, { i18n, localVue })
    expect(wrapper).toMatchSnapshot()
  })
})
