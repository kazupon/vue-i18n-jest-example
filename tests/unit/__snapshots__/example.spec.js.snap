// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloI18n.vue renders i18n custom blocks 1`] = `<p>Hello i18n in SFC!</p>`;
